api/v2.0/swagger.yaml

```diff
@@ -9244,4 +9320,58 @@ definitions:
         type: array
         items:
           $ref: '#/definitions/ScanDataExportExecution'
-        description: The list of scan data export executions
\ No newline at end of file
+        description: The list of scan data export executions
+  WorkerPool:
+    type: object
+    description: the job pool
```

Comment:
is it a WorkerPool or Job Pool? What is the differences in name and description.

Reply:
Updated